### ISSUE TITLE
Upper bound for rain delay too low

### ISSUE THREAD

**Issue by yolani** [Barma-lej/halandroid#51]
Your upper range for https://github.com/Barma-lej/halandroid/blob/1194d9a81ca90ead75560b5659b5c4acd6012d66/packages/landroid.yaml#L363 is set to low. For my WR165E I can set rain delay in the app to max 24h. 
Setting it to anything > 300 minutes causes an error in HA.

**Comment by Barma-lej** [comment]
Can't confirm 24h
![Screenshot_20220427-111248_Landroid](https://user-images.githubusercontent.com/34010886/165485553-09d65dd6-e76c-4580-ba40-caa10b05de55.jpg)
 I also can't increase rain delay because [landroid integration](https://github.com/MTrab/landroid_cloud) has the limit up to 300 min
https://github.com/MTrab/landroid_cloud/blob/676b4b0fce5959ad161ca7fc2a8335fd9d5fe8fb/custom_components/landroid_cloud/services.yaml#L94

**Comment by yolani** [comment]
Interesting, what Modell do you have? My app let's me increase the delay to 23h and 30m to be precise![image](https://user-images.githubusercontent.com/219931/165504584-02a296e1-1895-4002-be57-99f27f6a354d.png)

**Comment by Barma-lej** [comment]
WR141E

**Comment by yolani** [comment]
Okay so I guess the WR14x and the WR16x models differ here. It's not a big thing, I can patch this locally.

Even if you could get the max_delay through the API I don't think you could use it to set the max of the input_number...

**Comment by Barma-lej** [comment]
And you also can't increase rain delay because [landroid integration](https://github.com/MTrab/landroid_cloud) has the limit up to 300 min 😉
 
https://github.com/MTrab/landroid_cloud/blob/676b4b0fce5959ad161ca7fc2a8335fd9d5fe8fb/custom_components/landroid_cloud/services.yaml#L94

**Comment by yolani** [comment]
But that's only a descriptive string and no real range boundary. Actually the Landroid integration receives my delay properly ![image](https://user-images.githubusercontent.com/219931/165605756-69382e0b-5081-427c-8ea1-ada1a64bdd75.jpeg)

**Comment by Mincka** [comment]
Same issue for WR165E, the limit through the mobile app is 12 hours.
`[homeassistant.components.automation.mower_sync_values] Mower sync values: Error executing script. Invalid data for call_service at pos 1: Invalid value for input_number.mower_raindelay: 720.0 (range 0.0 - 300.0)`

I had a look at MTrab/landroid_cloud and MTrab/pyworxcloud and as @yolani said, I did not find a limitation of the integration. I think it's fine to change it here so I did a PR that fits @yolani's model.
